ansible/roles/php/defaults/main.yml

```diff
@@ -756,7 +756,8 @@ php__apt_preferences__dependent_list:
 
   - packages: [ 'php', 'php5', 'php5*', 'php7*', 'dh-php', 'php-*',
                 'libpcre2-8-0', 'libpcre3', 'libzip4', 'libpcre16-3',
-                'libpcre32-3', 'libpcrecpp0v5', 'libpcre3-dev' ]
+                'libpcre32-3', 'libpcrecpp0v5', 'libpcre3-dev',
+                'libapache2-mod-php' ]
```

Comment:
You also need to include `libapache2-mod-php*` which will cover versioned packages in later Debian releases. See https://packages.debian.org/search?keywords=libapache2-mod-php

Reply:
Yeah, good point !